bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -1111,14 +1108,7 @@ public void sendAnnouncement(Device device, String text, @Nullable String bodyTe
         } else {
             content.display.title = title;
         }
-        content.display.body = text;
-        if (text.startsWith("<speak>") && text.endsWith("</speak>")) {
-            content.speak.type = "ssml";
```

Comment:
Is this superfluous? It was set before but not in the new version.

Reply:
You are right again, this is missing.